lib/summarizer.py

```diff
@@ -184,10 +184,10 @@ def _SummarizeSameMask(nets):
       # look for pair net, but keep index handy
       for pair_net_index, pair_net in enumerate(current_nets):
         current_address, current_netmask = current_net
-        pair_address, _ = pair_net
+        pair_address, pair_netmask = pair_net
         xored_address = current_address ^ pair_address
         # check if networks have exactly one bit difference, or are "a pair"
```

Comment:
Can you update this comment to explain what you are doing here? It explains the check previous to your change but it was not expanded to include what it does now.